tr_barcode/tr_barcode.py

```diff
@@ -33,6 +32,9 @@
     _logger.warning("ERROR IMPORTING REPORT LAB")
```

Comment:
no need, the try-except blocks are ignored already. Just put `_logger` after the last top-level import.